### PR TITLE
fix: login icon style

### LINKED ISSUE #1011: When user see login form, cloud icon has different style
**Describe the bug**
When user see login form, cloud icon has different style

**To Reproduce**

Go to login
see error

**Screenshots**
image

### PR DESCRIPTION
**Pull request by parkhojeong**
fix login form cloud icon element style

before fix
image

after fix
image

this pr close #1011

### CODE REVIEW

File: src/components/backend-ai-login.ts [hunk 1]
```diff
@@ -1143,7 +1143,7 @@ export default class BackendAILogin extends BackendAIPage {
             <form>
               <fieldset>
                 <div class="horizontal layout" id="id_api_endpoint_container" style="display:none;">
-                  <mwc-icon-button id="endpoint-button" icon="cloud_queue" style="margin-left:5px;" @click="${() => this._toggleEndpoint()}"></mwc-icon-button>
+                  <mwc-icon-button id="endpoint-button" icon="cloud_queue" style="background-color: rgb(250, 250, 250);" @click="${() => this._toggleEndpoint()}"></mwc-icon-button>
```
Comment (lizable) [inline src/components/backend-ai-login.ts]: How about adding `3px` margin on the left side of this cloud icon button?
It looks like more aligned with the icons above when you add a little bit of margin.

**Review by lizable — CHANGES_REQUESTED** [review]
I just add a comment for a slight change.

File: src/components/backend-ai-login.ts (hunk 1 above)
Comment (parkhojeong) [reply src/components/backend-ai-login.ts]: `padding-left:3px` is better than `margin-left:3px`. thanks to your comment. LGTM 👍 
margin-left 
image


padding-left
image

**Review by lizable — APPROVED** [review]
LGTM.

**Pull request merged**